common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -487,6 +487,10 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
    * in the underlying Hadoop configuration.
    */
   public static enum ConfVars {
+    MSC_CACHE_ENABLED("hive.metastore.client.cache.enabled", true,
+            "This property enables a Caffeiene LoadingCache for Metastore client"),
+    MSC_CACHE_MAX_SIZE("hive.metastore.client.cache.maxSize", 1_000_000_000,
```

Comment:
Can we also set the default lower for the tests? In particular, you can do that by adding `hive.metastore.client.cache.maxSize` property with value of `10MB` to `data/conf/hive-site.xml`, `data/conf/llap/hive-site.xml` and `data/conf/rlist/hive-site.xml`.

Reply:
1. Can I just add `10MB`, or do I have to add the actual number `10000000` ? 
2. Do I also add similar property in corresponding `hivemetastore-site.xml`?

Reply:
1. You can add 10MB indeed.
2. I am not sure about the initialization process. I think it should be taken from those `hive-site.xml` but you can certainly check that the value is correctly set.